pkg/TIPA.py

```
@@ -11,34 +11,57 @@
+from win32gui import GetForegroundWindow, GetWindowText
 # pylint: enable=no-name-in-module
-pytesseract.pytesseract.tesseract_cmd = r"C:\Program Files\Tesseract-OCR\tesseract.exe"
 
+pytesseract.pytesseract.tesseract_cmd = r"D:\Program Files\Tesseract-OCR\tesseract.exe"
+
+logging.basicConfig(level=logging.INFO)
+
+
+class ProcessManager2(threading.Thread):
+    def __init__(self):
+        super(ProcessManager2, self).__init__(name="ProcessManagerThread")
```

Ruff (0.8.0)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)